script/translations/upload.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+"""Merge all translation sources into a single JSON file."""
+import json
+import os
+import pathlib
+import re
+import subprocess
+
+from .const import DOCKER_IMAGE, INTEGRATIONS_DIR, PROJECT_ID
+from .error import ExitApp
+from .util import get_current_branch, get_lokalise_token
+
+FILENAME_FORMAT = re.compile(r"strings\.(?P<suffix>\w+)\.json")
+LOCAL_FILE = pathlib.Path("build/translations-upload.json").absolute()
+CONTAINER_FILE = "/opt/src/build/translations-upload.json"
+LANG_ISO = "en"
+
+
+def run_upload_docker():
+    """Run the Docker image to upload the translations."""
+    print("Running Docker to upload latest translations.")
+    run = subprocess.run(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5338343)